types/stylis/index.d.ts

```diff
@@ -0,0 +1,147 @@
+// Type definitions for stylis 4.0
+// Project: https://github.com/thysultan/stylis.js
+// Definitions by: Gen Hames <https://github.com/heyheyhello>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
```suggestion

export as namespace stylis;

```
you folks probably want to support UMD, as per cocmment on the README:
https://github.com/thysultan/stylis.js#installation
If accepted, this would work:
```ts
stylis.serialize(stylis.compile(`h1{all:unset}`), stylis.stringify)
// "h1{all:unset;}"
```

Reply:
Great catch thanks! Adding that now